src/genie/libs/parser/ironware/tests/ShowIPOSPFInterfaceBrief/cli/equal/documentation_output_expected.py

```diff
@@ -0,0 +1,13 @@
+expected_output = {
+  'total': 1,
+  'interfaces': {
+    'ethernet1/2': {
```

Comment:
converting from `eth 1/2` to `ethernet1/2`?

Reply:
@tahigash Correct. Expanding interface names in the parsers. Is this non standard?

Reply:
this is ok. we are doing the same on XE/XR/NXOS. just wanted to confirm.